lusidtools/apps/upsert_instruments.py

```diff
@@ -71,20 +60,16 @@ def load_instruments(args):
         else [],
     )
 
-    total_success = sum(
-        [
-            len(resp.values.keys())
-            for resp in instruments_response["instruments"]["success"]
-        ]
-    )
-    total_failed = sum(
-        [
-            len(resp.failed.keys())
-            for resp in instruments_response["instruments"]["success"]
-        ]
-    )
-    logging.info(f"Success: {total_success}/{len(instruments)}")
-    logging.info(f"Fail:    {total_failed}/{len(instruments)}")
+    succ, errors, failed = cocoon_printer.format_instruments_response(instruments_response)
+    logging.info(f"number of successful upserts: {len(succ)}")
+    logging.info(f"number of failed upserts    : {len(failed)}")
+    logging.info(f"number of errors            : {len(errors)}")
+
+    if args["display_full_response"]:
```

Comment:
If you change the argument name in the command line, make sure to change this too.